src/documents/APIChangesFrom2_0To3_0.md

```diff
@@ -19,10 +19,56 @@ This section describes major APIs that were in common use. For information about
 
 ### Preferences Framework ###
 
+#### Model Changes ####
+
+##### Reset ######
+By default, reset will only reset the `preferences` model path. Other model values will remain unchanged. If you'd like to also have these paths changed, add a listener to the `beforeReset` event to execute a fireChangeRequest for the model paths you need to reset. (See: ArrowScrolling.js)
```

Comment:
Might be helpful to add a link for `ArrowScrolling.js` so ppl don't have to look around. It would be more helpful to have this link directly point to the lines that has the example of using `beforeReset` event, but considering those lines will likely change over time by new changes to this file, probably better not to include line numbers.